index.html

```diff
@@ -7,6 +7,6 @@
   <title>Welcome to Potionfy</title>
 </head>
 <body>
-  
+  <h1>Welcome to Potionfy! Enjoy your stay.<h1>	  
```

Comment:
Could you remove the "Enjoy your stay.", string and make the "Welcome the Potionfy!" string red?

Reply:
Sure, can do!

Reply:
Please, change the code as per my comments on the review.